UPDATING.md

```diff
@@ -216,6 +216,28 @@ The previous option used a colon(`:`) to split the module from function. Now the
 
 The change aims to unify the format of all options that refer to objects in the `airflow.cfg` file.
 
+#### Unify user session lifetime configuration
+
+In previous version of Airflow user session lifetime could be configured by
+`session_lifetime_days` and `force_log_out_after` options. In practise only `session_lifetime_days`
+had impact on session lifetime, but it was limited to values in day.
+In Airflow 2.0 we have removed mentioned options and introduced new `session_lifetime_minutes`
```

Comment:
Hi @michalmisiewicz . Thanks for the update. 

All look good to me other than a last point: this change should be put under section `Airflow Master`, rather than `Airflow 2.0.0a1`. The release manager(s) will decide when and which version to include this.

May you please make this change? Please also remember to rebase to the latest master. After that, I think we are good to merge.

Thanks!

Reply:
Please also remove "In Airflow 2.0", given in which version we should include this change is not determined yet. Thanks.